ios/FluentUI.Demo/FluentUI.Demo/Demos/PreviewCardDemoController.swift.orig

```diff
@@ -0,0 +1,50 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+import SwiftUI
+
+class PreviewCardDemoController: DemoTableViewController {
+    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
+    }
+    required init?(coder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+    override func viewDidLoad() {
+        super.viewDidLoad()
+<<<<<<< HEAD
+        let previewFlatCard = MSFPreviewCard(theme: nil, elevated: false)
+        let previewElevatedCard = MSFPreviewCard(theme: nil, elevated: true)
+        let previewFlatCardView = previewFlatCard.view
+        let previewElevatedCardView = previewElevatedCard.view
+        view.addSubview(previewFlatCardView)
+        view.addSubview(previewElevatedCardView)
+        previewFlatCardView.translatesAutoresizingMaskIntoConstraints = false
+        previewElevatedCardView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            previewFlatCardView.topAnchor.constraint(equalTo: view.topAnchor, constant: Constants.constantOffset),
+            previewElevatedCardView.topAnchor.constraint(equalTo: previewFlatCardView.bottomAnchor, constant: Constants.constantOffset),
+            previewFlatCardView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: Constants.constantOffset),
+            previewElevatedCardView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: Constants.constantOffset)
+=======
```

Comment:
Looks like there are some git conflicts to resolve?

Reply:
Yeah, ran into a bunch of git issues and currently fixing the conflicts

Reply:
> Looks like there are some git conflicts to resolve?

Yeah, ran into a bunch of git issues and currently fixing the conflicts